db/migrate/20150929152345_move_arf_reports_to_reports_table.rb

```diff
@@ -3,60 +3,15 @@ class MoveArfReportsToReportsTable < ActiveRecord::Migration
   # rubocop:disable Metrics/MethodLength
   # rubocop:disable Metrics/AbcSize
   def up
-    old_arf_reports = execute("SELECT * FROM foreman_openscap_arf_reports;")
-
-    #select only reports with existing host
-    old_arf_reports = old_arf_reports.select do |item|
-      asset = ForemanOpenscap::Asset.find item['asset_id']
-      !asset.host.nil? && asset.assetable_type = "Host::Base"
```

Comment:
oh, you've fixed another bug by this, I guess the one `=` was not intentional here :-)